content/influxdb/v2.3/influxdb-templates/use.md

```diff
@@ -13,213 +13,778 @@ related:
+              "kind": "Foo",
+              "metadata": {
+                "name": "unruffled-benz-d4d007"
+              },
+              "spec": {
+                "color": "#326BBA",
+                "name": "inputs.cpu"
+              }
+            }
+          ]}
+      }
+EOF
+```
+
+InfluxDB responds with a `422` status code and a response body similar to the following:
+
+{{% truncate %}}
+
+```json
+{
+  "sources": [],
+  "stackID": "",
+  "diff": {
+    "buckets": [],
+    "checks": [],
+    "dashboards": [],
+    "labels": [],
+    "labelMappings": [],
+    "notificationEndpoints": [],
+    "notificationRules": [],
+    "tasks": [],
+    "telegrafConfigs": [],
+    "variables": []
+  },
+  "summary": {
+    "buckets": [],
+    "checks": [],
+    "dashboards": [],
+    "notificationEndpoints": [],
+    "notificationRules": [],
+    "labels": [],
+    "labelMappings": [],
+    "missingEnvRefs": null,
+    "missingSecrets": null,
+    "summaryTask": [],
+    "telegrafConfigs": [],
+    "variables": []
+  },
+  "errors": [
+    {
+      "kind": "Foo",
+      "fields": [
+        "root",
+        "kind"
+      ],
+      "idxs": [
+        0,
+        null
+      ],
+      "reason": "unsupported kind provided"
+    }
+  ],
+  "code": "unprocessable entity",
+  "message": "unprocessable entity"
+}
+```
+
+{{% /truncate %}}
+
+Given that the template contains an invalid InfluxDB resource type (`"Foo"`),
+the template fails validation and the response contains the following:
+
+- empty resource diffs
+- list of errors
+- error code and message
+
+## Apply templates to an InfluxDB instance
+
+When you apply a template, you specify the organization that owns the template resources.
+InfluxDB validates and then installs the template to create and update resources in the organization.
+If you apply a template without providing a **stack ID**, InfluxDB initializes a new stack with all new resources.
+
+To learn how to use InfluxDB stacks, see [InfluxDB stacks](/influxdb/v2.3/influxdb-templates/stacks/).
+
+After applying the template, InfluxDB responds with the following:
+
+- the stack ID for the stack that was created or updated
+- the diff of added and removed resources
+
+{{% note %}}
+When applying templates, you must pass a token that has `write` permission for the
+resource types configured in the template.
+{{% /note %}}
+
+Use the `influx` CLI or `/api/v2` API to apply templates to an InfluxDB instance.
+
+- [Apply templates with the CLI](#apply-templates-with-the-cli)
+- [Apply templates with the API](#apply-templates-with-the-api)
+
+## Apply templates with the CLI
+
+Use the [`influx apply` command](/influxdb/v2.3/reference/cli/influx/apply/) to install templates to your InfluxDB organization.
+You can apply templates stored in your local file system or from URLs.
+
+The following code sample uses `influx apply` with an
+[all-access token]({{< latest "influxdb" >}}/security/tokens/) to
```

Comment:
We should use the `latest` shortcode to reference the same product/version. This will lead to inadvertent version switches when a new version gets released. 

```suggestion
[all-access token](/influxdb/v2.3/security/tokens/) to
```